modules/nw-ovn-k-adminnetwork-policy.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn-k-network-policy.adoc
+
+:_mod-docs-content-type: <Concept>
+[id="AdminNetworkPolicy"_{context}"]
+= AdminNetworkPolicy
+
+`AdminNetworkPolicy` (ANP) is a cluster-scoped custom resource definitions (CRDs). As an {product-title} administrator, you can use `AdminNetworkPolicy` to secure your cluster in the following ways:
+
+* You can create cluster-scoped policies prior to namespaces being created.
+
+* You can create deny and allow traffic rules at a cluster level that are non-overridable by users in the cluster.
+
+The key difference between `AdminNetworkPolicy` and `NetworkPolicy` is that ANP provides cluster scoped policy for administrators while `NetworkPolicy` provides namespaced scoped policy for developers.
+
+An ANP allows administrators to specify:
+
+* A `subject` that consists of a set of pods that is selected by the policy
+
+* A list of ingress rules to be applied for all ingress traffic towards the `subject`.
+
+* A list of egress rules to be applied for all egress traffic from the `subject`.
+
+//this content may need to be moved to the assembly to help discuss ACLs with the image. This would provide a more detailed picture of the integration between ANP and BANP with NP API.
+If traffic matches an ANP rule, the rules in that ANP will be evaluated first. If the match is an ANP `allow` or `deny` rule, any existing `NetworkPolicies` and `BaselineAdminNetworkPolicy` (BANP) in the cluster will be intentionally skipped from evaluation. If the match is an ANP `pass` rule, then evaluation moves from tier 1 of the ACLs to tier 2 where `NetworkPolicy` is evaluated.
+
+[discrete]
+[id="AdminNetworkPolicy-example"]
+== AdminNetworkPolicy example
+`AdminNetworkPolicy` is a `TechnologyPreviewNoUpgrade` feature that can be enabled on test clusters that are not in production. The feature is behind the flag `enable-admin-network-policy`. See "Enabling features using feature gates" in the Additional resources of this section for more information on feature gates and `TechnologyPreviewNoUpgrade` features.
+
+.Example YAML file for ANP
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: priority-50-example <1>
```

Comment:
I would give this a better name that suits the purpose of the policy rather than repeating its priority number?